ui/src/components/ModalVhostEdit.vue

```diff
@@ -352,6 +352,155 @@ select {
+                {{$t('virtualhost.php71_not_installed')}}.
+              </div>
+
+              <!-- php version -->
+              <div v-bind:class="['form-group', vErrors.PhpRhVersion ? 'has-error' : '']">
+                <label
+                  class="col-sm-3 control-label"
+                  v-bind:for="id + '-certs'"
+                >{{$t('virtualhost.php_version')}}</label>
+                <div class="col-sm-9">
+                  <select
+                    type="text"
+                    v-model="PhpRhVersion"
+                    class="combobox form-control col-sm-9"
+                  >
+                    <option value="default">{{$t('virtualhost.default_php_version')}}</option>
+                    <option value="php71">{{$t('virtualhost.php71_version')}}</option>
+                    <option value="php72">{{$t('virtualhost.php72_version')}}</option>
+                  </select>
+                  <span
+                    v-if="vErrors.PhpRhVersion"
+                    class="help-block"
+                  >{{ vErrors.PhpRhVersion }}</span>
+                </div>
+              </div>
+
+              <!-- Cutomise php for default php version -->
+              <div v-bind:class="['form-group', vErrors.phpCustomSettings ? 'has-error' : '']">
+                <label
+                  class="col-sm-3 control-label"
+                  v-bind:for="id + '-defaultPhpSettings'"
+                >{{$t('virtualhost.phpCustomSettings')}}</label>
+                <div class="col-sm-9">
+                    <toggle-button
+                      class="min-toggle"
+                      :width="40"
+                      :height="20"
+                      :color="{checked: '#0088ce', unchecked: '#bbbbbb'}"
+                      :value="phpCustomSettings == 'enabled'"
+                      :sync="true"
+                      @change="phpCustomSettings == 'enabled' ? phpCustomSettings = 'disabled' : phpCustomSettings = 'enabled'"
+                    />
+                  <span
+                    v-if="vErrors.phpCustomSettings"
+                    class="help-block"
+                  >{{ vErrors.phpCustomSettings }}</span>
+                </div>
+              </div>
+
+              <!-- slider -->
+              <div v-if="phpCustomSettings === 'enabled'">
+                <div :class="['form-group', vErrors.MaxExecutionTime ? 'has-error' : '']">
+                    <label class="col-sm-3 control-label">{{$t('virtualhost.MaxExecutionTime')}}
+                        <doc-info
+                          :placement="'top'"
+                          :title="$t('virtualhost.MaxExecutionTime')"
+                          :chapter="'MaxExecutionTime'"
+                          :inline="true"
+                        ></doc-info>
+                    </label>
+                    <div class="col-sm-9">
+                        <div v-if="MaxExecutionTime > 0">{{MaxExecutionTime + ' '+ $t('virtualhost.seconds')}}</div>
```

Comment:
The vue-i18n extension has support for both formatting and pluralization

- https://kazupon.github.io/vue-i18n/guide/formatting.html
- https://kazupon.github.io/vue-i18n/guide/pluralization.html

For instance 
- (formatting) https://github.com/NethServer/nethserver-samba/blob/6bc69adc70f070ebb2e0c8379bfb1b5aba42b3dd/ui/src/components/SharedFolderEditModal.vue#L78
- (pluralization) - can't find it

Reply:
pluralization done